app/assets/stylesheets/color_themes/_color_theme_override_dark.scss

```diff
@@ -0,0 +1,170 @@
+body {
+  .navbar.navbar-fixed-top #user_menu .dropdown-menu > li > a {
+    color: $text-color;
+    &:hover { color: $white; }
+  }
+
+  .publisher {
+    .mentions-input-box { background-color: $gray; }
+    form {
+      #publisher_textarea_wrapper { background-color: $gray; }
+      .btn.btn-link.question_mark:hover .entypo-cog { color: $gray-light; }
+    }
+
+    .write-preview-tabs > li.active * { color: $text-color; }
+    .md-preview { background-color: $gray; }
+    .md-cancel:hover .entypo-cross { color: $gray-light; }
+    .publisher-buttonbar .btn.btn-link:hover i { color: $gray-light; }
+  }
+
+  .aspect_dropdown li a .text { color: $dropdown-link-color; }
+
+  .info .tag { background-color: $gray-light; }
+
+  .poll_form .progress {
+    background-color: $gray-dark;
+    .bar { background-color: $gray-light; }
+  }
+
+  .stream-element .collapsible {
+    .markdown-content hr { border-top: 1px solid $hr-border; }
+    .expander {
+      @include linear-gradient(transparent, $gray-light, 0%, 95%);
+      border-bottom: 2px solid $gray-light;
+      color: $text-color;
+      text-shadow: 0 0 7px $black;
+    }
+  }
+
+  code,
+  pre {
+    background-color: $gray-dark;
+    border: 1px solid $border-medium-grey;
+    color: $text-color;
+  }
+
+  pre code { border: 0; }
+
+  @import 'highlightjs/darcula';
+
+  #single-post-content .head {
+    #post-info .author { color: lighten($gray-lighter, 27%); }
+    #single-post-actions i.entypo-heart.red:hover { color: $red; }
+  }
+
+  .opengraph a { color: lighten($gray-lighter, 27%); }
+
+  .tag:hover { background-color: desaturate(darken($blue, 35%), 20%); }
+
+  #profile_container .profile_header {
+    #author_info #sharing_message.entypo-check { color: lighten($green, 10%); }
+  }
+
+  #invitationsModal #email_invitation { border-top: 1px dashed $gray-light; }
+
+  #contacts_container #people_stream.contacts .stream-element.in_aspect {
+    background-color: $state-success-bg;
+    border-left: 3px solid darken($state-success-bg, 10%);
+  }
+
+  .left-navbar #tags_list {
+    .as-list {
+      color: $text-color;
+
+      em {
+        background-color: lighten($background-blue, 10%);
+        color: $text-color;
+      }
+    }
+
+    .as-result-item.active { color: $text-color; }
+  }
+
+  #faq .question {
+    background-color: $gray-dark;
+    a.toggle { color: $gray-lighter; }
+    &.collapsed { border: 2px solid $gray-dark; }
+    &.opened {
+      border: 2px solid darken($green, 10%);
+      h4 { background-color: darken($green, 10%); }
+    }
+
+    .answer { background-color: $gray; }
+  }
+
+  #welcome-to-diaspora { background: $orange; }
+
+  .block-form fieldset .form-control:focus { border-color: $input-border; }
+
+  &.page-registrations.action-new,
+  &.page-registrations.action-create {
+    .ball { filter: invert(100%); }
+  }
+
+  .spinner { border-color: $gray-light transparent $gray-light $gray-light; }
```

Comment:
Shorthands of length `4` are not allowed. Value was `$gray-light transparent $gray-light $gray-light`